### PR TITLE
set ovirt disk active default  (#57464)

### PR DESCRIPTION
**Pull request by mnecas**
Backport of https://github.com/ansible/ansible/pull/57464 

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

ovirt
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: lib/ansible/modules/cloud/ovirt/ovirt_disk.py [hunk 1]
```diff
@@ -684,6 +685,9 @@ def main():
         ret = None
         # First take care of creating the VM, if needed:
         if state in ('present', 'detached', 'attached'):
+            # Always activate disk when its being created
```
Comment (mwperina) [inline lib/ansible/modules/cloud/ovirt/ovirt_disk.py:706]: You need to define vm_service:
```
vm_service = get_vm_service(connection, module)
```

Comment (mnecas) [reply lib/ansible/modules/cloud/ovirt/ovirt_disk.py:706]: Yeah sorry it was defined already in devel i have not noticed that it wasnt in stable28

**Review by mwperina — APPROVED** [review]
shipit

**Comment by abadger** [comment]
Merged for the 2.8.2 release

**Pull request merged**